sdk/identity/Azure.Identity.Broker/CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.1.0 (2024-04-09)
 
 ### Other Changes
 
```

Comment:
This is technically a breaking change from last beta ?

Reply:
yes, but I didn't think we categorize changes in betas as breaking changes.

Reply:
Correct. It's implied that we'll introduce breaks in Betas.

Reply:
as a user, if I didn't try any beta versions, I see this change in the minor version bump. it comes across as a breaking change from last stable release.

Reply:
It's additive from the previous stable release.